pages/docs/react/latest/migrate-from-v3.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title:
+description: "Migrate from JSX v3"
+canonical: "/docs/react/latest/jsx-v3-v4"
+---
+
+# Migrate from JSX v3
+
+JSX v4 introduces a new idiomatic record-based representation of components which is incompatible with v3. Because of this, either the entire project or dependencies need to be compiled in V4 mode, or some compatibility features need to be used to mix V3 and V4 in the same project. This page describes how to migrate from v3 to v4.
```

Comment:
```suggestion
JSX v4 introduces a new idiomatic record-based representation of component props which is incompatible with v3. Because of this, either the entire project or dependencies need to be compiled in V4 mode, or some compatibility features need to be used to mix V3 and V4 in the same project. This page describes how to migrate from v3 to v4.
```